### PR TITLE
GAP-2255: Download submission for individual or non-limited applicant

### PR DESCRIPTION
**Pull request by ConorFayleTCO**
[GAP-2255](https://technologyprogramme.atlassian.net/browse/GAP-2255)

add conditional logic for content, add optional get for conditional questions

[GAP-2255]: https://technologyprogramme.atlassian.net/browse/GAP-2255?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/java/gov/cabinetoffice/gap/testdata/TestData.java [hunk 1]
```diff
@@ -18,155 +18,446 @@
...
+    public static final SubmissionSection ORGANISATION_DETAILS_SECTION_SUBMISSION_INDIVIDUAL = SubmissionSection.builder()
+            .sectionId("ORGANISATION_DETAILS")
+            .sectionTitle("Your details")
+            .sectionStatus(SubmissionSectionStatus.COMPLETED)
+            .questions(Arrays.asList(
+                    V2_ORG_TYPE_SUBMISSION_QUESTION_INDIVIDUAL,
+                    V2_ORG_NAME_SUBMISSION_QUESTION,
+                    V2_ORG_ADDRESS_SUBMISSION_QUESTION
+            ))
+            .build();
```
Comment (ConorFayleTCO) [inline src/main/java/gov/cabinetoffice/gap/testdata/TestData.java]: this has been added

File: src/main/java/gov/cabinetoffice/gap/testdata/TestData.java [hunk 2]
```diff
@@ -18,155 +18,446 @@
...
+            .sectionTitle("Your organisation")
             .sectionStatus(SubmissionSectionStatus.COMPLETED)
-            .questions(Arrays.asList(V2_ORG_TYPE_SUBMISSION_QUESTION, V2_ORG_NAME_SUBMISSION_QUESTION,
-                    V2_ORG_ADDRESS_SUBMISSION_QUESTION, V2_ORG_COMPANIES_NO_SUBMISSION_QUESTION, V2_ORG_CHARITY_NO_SUBMISSION_QUESTION))
+            .questions(Arrays.asList(
+                    V2_ORG_TYPE_SUBMISSION_QUESTION_NON_LIMITED,
+                    V2_ORG_NAME_SUBMISSION_QUESTION,
+                    V2_ORG_ADDRESS_SUBMISSION_QUESTION
+            ))
             .build();
```
Comment (ConorFayleTCO) [inline src/main/java/gov/cabinetoffice/gap/testdata/TestData.java]: this has been added

File: src/main/java/gov/cabinetoffice/gap/testdata/TestData.java [hunk 3]
```diff
@@ -18,155 +18,446 @@
...
+
+    public static final Submission V2_SUBMISSION_LIMITED_COMPANY = Submission
+            .builder()
+            .legalName("Test Org Name v2")
+            .schemeName("v2Scheme Name")
+            .gapId("GAP-LL-20220927-00001")
+            .submittedDate(Instant.parse("2022-02-15T18:35:24.00Z"))
+            .sections(V2_SUBMISSION_SECTIONS_LIST)
+            .schemeVersion(2)
+            .build();
```
Comment (ConorFayleTCO) [inline src/main/java/gov/cabinetoffice/gap/testdata/TestData.java]: this has been renamed

File: src/main/java/gov/cabinetoffice/gap/testdata/TestData.java [hunk 4]
```diff
@@ -18,155 +18,446 @@
...
+
+    public static final Submission V2_SUBMISSION_INDIVIDUAL = Submission
+            .builder()
+            .legalName("Test Individual v2")
+            .schemeName("v2Scheme Name")
+            .gapId("GAP-LL-20220927-00001")
+            .submittedDate(Instant.parse("2022-02-15T18:35:24.00Z"))
+            .sections(V2_SUBMISSION_SECTIONS_LIST_INDIVIDUAL)
+            .schemeVersion(2)
+            .build();
```
Comment (ConorFayleTCO) [inline src/main/java/gov/cabinetoffice/gap/testdata/TestData.java:518]: this has been added

**Comment by ConorFayleTCO** [comment]
refactored TestData.java because it was difficult to read, have added comments on lines added/modified

**Review by paul-lawlor-tco — APPROVED** [review]

**Review by jgunnCO — APPROVED** [review]

**Pull request merged**